src/Utils/obiUtils.tsx

```diff
@@ -62,12 +62,12 @@ export interface TranscriptActionCall {
     type: string
     actionName: string
     actionInput: TranscriptActionInput[]
-    actionOutput: TranscriptActionOutput[]
+    actionOutput: ActionOutput[]
```

Comment:
nit - I might name this ObiActionOutput to namespace it / give context.  As it's being exported now "ActionOutput" seems a big generic.

Reply:
Done.